qualtran/resource_counting/_bloq_counts.py

```diff
@@ -174,36 +232,59 @@ def _is_nonzero(v):
                 return True
             return maybe_nonzero
 
-        return {k: v for k, v in d.items() if _is_nonzero(v)}
+        def _keep(key, value) -> bool:
+            if key == 'binned_rotation_epsilons':
+                return value
+            if key == 'eps_bin_prec':
+                # rotations non-empty
+                return len(self.binned_rotation_epsilons) > 0
+            return _is_nonzero(value)
+
+        return {k: v for k, v in d.items() if _keep(k, v)}
+
+    @staticmethod
+    def rotation_t_cost(eps: SymbolicFloat) -> SymbolicInt:
+        """T-cost of a single Z rotation with precision `eps`.
+
+        References:
+            [Efficient synthesis of universal Repeat-Until-Success circuits](https://arxiv.org/abs/1404.5320)
+            Bocharov et. al. 2014. Page 4, Paragraph "Simulation Results."
+        """
+        return ceil(1.149 * log2(1.0 / eps) + 9.2)
+
+    @property
+    def rotation_to_t(self) -> SymbolicInt:
```

Comment:
I'm not sure I like this as a property. Even its name suggests an "action". It's name also suggests that it takes rotations and turns it to t. It sounds very similar to the static method name `rotation_t_cost` if I didn't see the code.

Suggest: `total_rotations_as_t()`

Reply:
done